src/utils/process-utils.ts

```
@@ -0,0 +1,53 @@
+/**
+ * Checks if a process with the given PID is still running.
+ *
+ * @param pid - Process ID to check.
+ * @returns True if the process is running, false otherwise.
+ */
+export async function isProcessAlive(pid: number): Promise<boolean> {
+	try {
+		const hideOutput = { shouldShowCommand: false, shouldStreamOutput: false };
+
+		await runPlatform({
+			darwin: async () => run("ps", ["-p", String(pid)], hideOutput),
```

ESLint

[error] Unsafe return of a value of type `Promise<any>`.

(ts/no-unsafe-return)